src/main/sources/VcfDataSource.js

```diff
@@ -91,13 +92,16 @@ function createFromVcfFile(remoteSource: VcfFile): VcfDataSource {
   return o;
 }
 
-function create(data: {url:string}): VcfDataSource {
+function create(data: Object): VcfDataSource {
```

Comment:
No worries - I, also, had to check the `flow-type` reference and try a few things before I get that right. Not the most obvious solution ;)